### ISSUE TITLE
R3: for first/last corresponding, why take majority rule?

### ISSUE THREAD

**Issue by cgreene** [greenelab/iscb-diversity-manuscript#51]
> p. 3, penultimate paragraph: if the breakdown is approximately 40/60 on first/last authors being corresponding, why take a majority rule?  Seems more rigorous to randomize the selection process according to this "weighted die"

The paragraph in question is:
> We performed further analysis on PMC authors to learn more about corresponding author practices. First, we developed and evaluated a method to infer a corresponding author when the coded corresponding status was not available. For papers with multiple authors and at least one corresponding author, the first author was corresponding 43% of the time, whereas the last author was corresponding 62% of the time. Therefore, we assumed the last author was corresponding when coded corresponding author status was not available (120 articles from PMC and all articles from PubMed).

**Comment by trangdata** [comment]
Because majority rule would allow us to assume corresponding authors correctly 60% of the time, while randomized selection 52% (0.6^2 + 0.4^2 = 0.52).

**Comment by cgreene** [comment]
We should make this point. If it comes up again (or maybe we should just do this now), we could also do the analysis using only last authors and only corresponding authors so that folks can "choose their own adventure" through whichever set of results they view as most trustworthy. Every way we've looked at these data the key results from the abstract have been consistent.

**Comment by trangdata** [comment]
We can make this point. I think we already did the best we can with the data. Tripling the analysis may be overwhelming for the reader. We can wait to see if it comes up again.

**Comment by trangdata** [comment]
Here's a potential response:
> An alternative method is to randomly select corresponding author (first or last) according this proportion. However, this method, in general, will have lower accuracy compared to our hard assignment of where the majority of corresponding authors fall. In other words, if we let $p$ denote the majority proportion ($0.5 < p < 1$) then $p^2 +(1-p)^2 - p = (2p-1)(p-1) < 0$, which implies $p^2 + (1-p)^2 < p$, i.e., random selection has lower chance to assume corresponding authors correctly.

**Comment by dhimmel** [comment]
Nice response and logic @trang1618! I don't think we need to put this in the manuscript as long as we make our response to reviewers public and mention it in the manuscript. Adding this to the manuscript is a distraction for the majority of readers. It's great that we've justified most of our choices and have records in our notebooks / issues / response to reviewers, and it's important to keep the manuscript focused and succinct.

> we could also do the analysis using only last authors and only corresponding authors so that folks can "choose their own adventure" through whichever set of results they view as most trustworthy

I think the results would be very similar. Remember it's only ~30% of papers where we use last author to infer corresponding status. We've made hundreds of design decisions, and we could run our analysis hundreds of time for every permutation of decisions, but obviously that's not a scalable or desirable approach.
